### PR TITLE
the-sz.PreyHelper version 1.13

### PR DESCRIPTION
**Pull request by the-sz**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115355&drop=dogfoodAlpha

### CODE REVIEW

**Comment by the-sz** [comment]
The application needs admin right to run. Is this the problem? But previous version 1.12 has the same requirement and there, the PR was passing.

What is the exact reason for "Validation-Installation-Error"?

**Comment by the-sz** [comment]
What is the exact reason for "Validation-Installation-Error"? The application needs admin right to run. Is this the problem? But previous version 1.12 has the same requirement and there, the PR was passing.

File: manifests/t/the-sz/PreyHelper/1.13/the-sz.PreyHelper.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: the-sz.PreyHelper
+PackageVersion: 1.13
```
Comment (Trenly) [inline manifests/t/the-sz/PreyHelper/1.13/the-sz.PreyHelper.installer.yaml:5]:
```suggestion
PackageVersion: "1.13"
```
Floating point number needs to be quoted to be compliant with YAML semantics

File: manifests/t/the-sz/PreyHelper/1.13/the-sz.PreyHelper.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: the-sz.PreyHelper
+PackageVersion: 1.13
```
Comment (Trenly) [inline manifests/t/the-sz/PreyHelper/1.13/the-sz.PreyHelper.locale.en-US.yaml:5]:
```suggestion
PackageVersion: "1.13"
```
Floating point number needs to be quoted to be compliant with YAML semantics

File: manifests/t/the-sz/PreyHelper/1.13/the-sz.PreyHelper.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: the-sz.PreyHelper
+PackageVersion: 1.13
```
Comment (Trenly) [inline manifests/t/the-sz/PreyHelper/1.13/the-sz.PreyHelper.yaml:5]:
```suggestion
PackageVersion: "1.13"
```
Floating point number needs to be quoted to be compliant with YAML semantics

**Review by Trenly — CHANGES_REQUESTED** [review]
> 2023-08-05 09:29:08.502 [FAIL] Installer failed security check. Url: https://the-sz.com/common/get.php?product=pr3y Result: 0x80004005

Probably need to wait a day or two for this to resolve

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Standard error:  Installation failed with exit code -1978335187 2023-08-05 09:29:08.502 [FAIL] Installer failed security check. Url: https://the-sz.com/common/get.php?product=pr3y Result: 0x80004005 

Please scan your systems to ensure that they're virus free. This might be a false positive, and we can rescan tomorrow.

**Comment by the-sz** [comment]
> Automatic Validation ended with:
> 
> > Standard error:  Installation failed with exit code -1978335187 2023-08-05 09:29:08.502 [FAIL] Installer failed security check. Url: https://the-sz.com/common/get.php?product=pr3y Result: 0x80004005
> 
> Please scan your systems to ensure that they're virus free. This might be a false positive, and we can rescan tomorrow.

https://www.virustotal.com/gui/file/f3840c6d7ca6f0098916c7b552b79288c53ae397220260d6e57f1c3cdbdd7e62/detection

only one false positive...

**Review by stephengillie — APPROVED** [review]

**Pull request merged**